Gemfile

```diff
@@ -7,56 +7,33 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```

Comment:
Are you sure you want to delete all the comments out of the gemfile? some are kind of useful